index.d.ts

```diff
@@ -0,0 +1,6 @@
+export type HookOptions = {
+  timeout?: number;
+  dependencies?: Array<String>;
+}
+export type HookReturnType = [(...fnArgs: any[]) => (Promise<any>), string, () => void];
+export function useWorker(fn: any, options: HookOptions): HookReturnType;
```

Comment:
Is `fn` really `any`? Or at least a function?

Reply:
Oh yes, thanks @Pigotz ;)